packages/core/android/src/main/kotlin/com/datadog/reactnative/JSONFileReader.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative
+
+import android.content.Context
+
+/**
+ * Utility for parsing JSON files from the app assets.
+ */
+class JSONFileReader {
```

Comment:
I believe there is no need to make it public.

```suggestion
internal class JSONFileReader {
```

Reply:
I also needed to make the constructor of `DdSdkNativeInitialization` internal for this to work, indeed that's a good point 👍